ecl/eclagent/eclagent.cpp

```diff
@@ -1950,6 +1951,25 @@ void EclAgent::doProcess()
             break;
         }
 
+        if (w->getState() == WUStateCompleted && getClusterType(clusterType)==ThorLCRCluster)
+        {
+            if (agentTopology->getPropBool("@analyzeWorkunit", true) && w->getDebugValueBool("analyzeWorkunit", true))
```

Comment:
The following is better - since it allows explicit overriding both ways:

`(w->getDebugValueBool("analyzeWorkunit", agentTopology->getPropBool("@analyzeWorkunit", true))
`

Reply:
Neat. That works better.